config/software/chefdk.rb

```diff
@@ -56,9 +56,9 @@
 
   # Perform multiple gem installs to better isolate/debug failures
   {
-    'chefspec'          => '4.2.0.beta.1',
+    'chefspec'          => '4.2.0',
     'fauxhai'           => '2.2.0',
-    'rubocop'           => '0.18.1',
+    'rubocop'           => '0.28.0',
     'knife-spork'       => '1.4.2',
```

Comment:
I believe @stevendanna asked whether we need to include `knife-spork` in the chef-dk because it slows down knife commands significantly - thoughts?